pyscf/lo/reg_emb.py

```diff
@@ -192,10 +198,26 @@ def get_virt_frozen(self, threshold=None):
             threshold = self.threshold
         return self.virt_loc_mos[:, numpy.abs(self.virt_frag_mo_ovlp) < threshold]
 
-    def kernel(self, threshold=None):
+    def print_thresholds(self, percentages=None):
+        frag_mo_ovlp = numpy.sort(numpy.concatenate([self.occ_frag_mo_ovlp, self.virt_frag_mo_ovlp]))[::-1]
+        self.log.info("Analyzing orbital inclusion with respect to threshold")
+        self.log.info("{}{}{}".format("-"*40, "-"*10, "-"*30))
+        self.log.info("{:>40}          {:>30}".format("Pct of full space", "Threshold"))
+        self.log.info("{}{}{}".format("-"*40, " "*10, "-"*30))
```

Comment:
just a stylistic question of why the spaces are in the center of this logger line?